src/components/windows-motion-controls.js

```diff
@@ -0,0 +1,382 @@
+var bind = require('../utils/bind');
+var registerComponent = require('../core/component').registerComponent;
+var controllerUtils = require('../utils/tracked-controls');
+var utils = require('../utils/');
+
+var debug = utils.debug('components:windows-motion-controls:debug');
+var warn = utils.debug('components:windows-motion-controls:warn');
+
+// TODO: Point to final glb assets once they are hosted. For now, place them in this hard coded directory.
+var MODEL_BASE_URL = '/examples/assets/models/controllers/wmr/';
+var MODEL_LEFT_FILENAME = 'left.glb';
```

Comment:
it might be better to consolidate names in an object:
````javascript
var MODEL_FILENAMES = { left: 'left.glb`, right: 'right.glb', default: 'universal.glb' }
````